bin/compile.js

```diff
@@ -152,6 +152,19 @@ var generate = function (callback) {
 		function (files) {
 			var data = null;
 			try {
+				data = manifest.compile(files.filter(function (file) {
+					return /(?:\.([^.]+))?$/.exec(file) === 'scss';
```

Comment:
Might be a good idea. There are no tests for the compile step at the moment, so might make sense to do this together.